test/app.js

```diff
@@ -1,103 +1,37 @@
-/* global artifacts contract before beforeEach it assert */
```

Comment:
I think eslint was complaining on this before, but not sure. Maybe we need to add a mocha or chai plugin?

Reply:
@0xGabi does this ring a bell?

Reply:
Yes, I believe it was after we include test in this [PR](https://github.com/aragon/aragon-react-boilerplate/pull/46)